style-guide/styles/wds-mixins/_button-group-theming.scss

```diff
@@ -0,0 +1,27 @@
+@mixin button-group-theming($color-border) {
+	> .wds-button {
+		&:not(.wds-is-secondary) {
+			&:not(:last-child) {
+				border-right: 1px solid $color-border;
```

Comment:
I moved completely those styles from button-group.scss to here, so we need to set style and width